### PR TITLE
Add the Psi Function for Geometric Kernel

### LINKED ISSUE #594: Add the Psi Function for Geometric Kernel
#### $\Psi(\alpha, \phi)$: Definitions

For the case when taking for the k-th droplet:

$$
\Psi(\alpha, \phi) = \frac{1}{(1/\tau_{pk}) + (1/\alpha) + (v_{pk}/\phi)} 
- \frac{v_{pk}}{2\phi \left((1/\tau_{pk}) + (1/\alpha) + (v_{pk}/\phi)\right)^2}
$$

### PR DESCRIPTION
**Pull request by Gorkowski**
Fixes #594

## Summary by Sourcery

Tests:
- Add tests for the psi function, covering various scenarios such as scalar and array inputs, invalid inputs, and edge cases.

### CODE REVIEW

**Comment by Gorkowski** [comment]
@sourcery-ai review

File: particula/dynamics/coagulation/turbulent_dns_kernel/psi_ao2008.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Psi function for the droplet collision kernel in the turbulent DNS model.
+"""
+from typing import Union
+import numpy as np
+from numpy.typing import NDArray
+
+from particula.util.validate_inputs import validate_inputs
+
+
+@validate_inputs(
```
[the remark this answers is not in the document]
Comment (Gorkowski) [reply particula/dynamics/coagulation/turbulent_dns_kernel/psi_ao2008.py:11]: @sourcery-ai issue
